services/gps2tsdb/Dockerfile

```diff
@@ -1,18 +1,34 @@
-FROM python:3
+# BUILDER
+FROM python:3 as builder
 
 WORKDIR /usr/src/app
 
 # Download latest listing of available packages:
 RUN apt-get -y update
-# Install a new package:
-RUN apt-get install -y apt-utils
-RUN apt-get install -y python3-dbus libdbus-glib-1-dev libdbus-1-dev
-RUN apt-get install -y libgirepository1.0-dev gcc libcairo2-dev pkg-config python3-dev gir1.2-gtk-3.0
-RUN ls /usr/lib/python3/dist-packages
+
+# Install packages required for DBUS dependencies
+
+# Needed for building dbus
+RUN apt-get install -y --no-install-recommends python3-dbus libdbus-glib-1-dev libdbus-1-dev
+
+# Needed for building PyGObject
+RUN apt-get install -y --no-install-recommends libgirepository1.0-dev gcc pkg-config python3-dev gir1.2-gtk-3.0
 
 COPY requirements.txt ./
-RUN pip install --no-cache-dir -r requirements.txt
+RUN pip install --user --no-cache-dir -r requirements.txt
+
+
+# RUNTIME
+FROM python:3 as runtime
+
+# Install runtime deps
+RUN apt-get -y update && apt-get install -y --no-install-recommends python3-dbus
```

Comment:
Should we `pip install` this since the python image does not use the Debian python package? ... I have no idea.